src/App.tsx

```diff
@@ -1,61 +1,112 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const titleInputId = 'title';
+const userInputId = 'user';
+const userSelectDefaultValue = '0';
+
+const allTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
 
 export const App = () => {
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
+  const [isUserEmpty, setIsUserEmpty] = useState(false);
+  const [maxTodoId, setMaxTodoId] = useState(
+    Math.max(...allTodos.map(todo => todo.id)),
+  );
+
+  const submitForm = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const titleInput
```

Comment:
create state for title with setter, and set value with onChange event

Reply:
I added title to state.